tests/openapi/client/openapi_source_check

```diff
@@ -85,6 +167,15 @@ if __name__ == "__main__":
         "list_source_with_redirect": list_source_with_redirect,
         "delete_source_failed": delete_source_failed,
         "delete_source_success": delete_source_success,
+        "start_relay_failed": start_relay_failed,
```

Comment:
how about using reflection to list all functions

Reply:
you mean something like `locals` or `inspect`? I think using this dynamic approach reduces the readability of the code ,so maybe it's better to keep it the way it is?